src/main/kotlin/com/cognifide/gradle/aem/environment/docker/DockerOptions.kt

```diff
@@ -0,0 +1,21 @@
+package com.cognifide.gradle.aem.environment.docker
+
+class DockerOptions {
+    var stackName = "local-setup"
+        set(value) {
+            if (value.isBlank()) {
+                throw DockerException("Stack name is required to run aemEnvUp (environment -> docker -> stackName)!")
```

Comment:
this `(environment -> docker -> stackName)!` convention is nowhere else used  I prefer regular precise, natural sentences with no impl details

Reply:
I removed this completely. In my opinion, this extension configuration was overengineering.